src/main/routes/routes.ts

```diff
@@ -80,10 +80,11 @@ export default function(app: Application): void {
   app.post('/search', app.locals.container.cradle.searchController.post);
   app.get('/sign-in', app.locals.container.cradle.signInController.get);
   app.post('/sign-in', app.locals.container.cradle.signInController.post);
-  app.get('/single-justice-procedure', app.locals.container.cradle.singleJusticeProcedureController.get);
   app.get('/view-option', app.locals.container.cradle.viewOptionController.get);
   app.post('/view-option', app.locals.container.cradle.viewOptionController.post);
-
+  app.get('/summary-of-publications', app.locals.container.cradle.summaryOfPublicationsController.get);
+  app.get('/file-publication', app.locals.container.cradle.flatFileController.get);
+  app.get('/list-publication', app.locals.container.cradle.listTypeController.get);
```

Comment:
should this be `/list-type`? it doesnt really render and publication and the controller is called list type, list publication seems like a random name